src/webrtc.ts

```diff
@@ -305,6 +305,8 @@ export class WebrtcExtension extends EventEmitter {
         this.handleRetract(msg.retract.sessionId);
       } else if (msg.accept) {
         this.handledIncomingRtcSession(msg.accept.sessionId);
+      } else if (msg.reject) {
+        this.handledIncomingRtcSession(msg.reject.sessionId);
```

Comment:
Sure, adding that comment makes sense. 🙂 

Regarding `handledIncomingRtcSession(sessionId: string, msg: unknown)`, TypeScript doesn't like `unknown` because of `msg.accept`. Is this a spot where `any` makes the most sense? Or is there another option?

Reply:
> Sure, adding that comment makes sense. 🙂
> 
> Regarding `handledIncomingRtcSession(sessionId: string, msg: unknown)`, TypeScript doesn't like `unknown` because of `msg.accept`. Is this a spot where `any` makes the most sense? Or is there another option?

I'm fine with using `any`.